packages/nimble-components/src/theme-provider/tests/theme-provider.spec.ts

```diff
@@ -16,6 +19,79 @@ describe('Theme Provider', () => {
         );
     });
 
+    describe('lang token', () => {
+        async function setup(
+            langValue = 'foo'
+        ): Promise<Fixture<ThemeProvider>> {
+            return fixture<ThemeProvider>(
+                html`<${themeProviderTag} lang="${langValue}">
+                    </${themeProviderTag}>`
+            );
+        }
+
+        let element: ThemeProvider;
+        let connect: () => Promise<void>;
+        let disconnect: () => Promise<void>;
+
+        afterEach(async () => {
+            await disconnect();
+        });
+
+        it('value is set to "foo" when theme provider lang attribute is assigned value "foo"', async () => {
+            ({ element, connect, disconnect } = await setup());
```

Comment:
This test and the next few confused me until I saw your PR comment about `"foo"` and `"bar"` being acceptable locale values. I don't think it's immediately obvious that those are valid, so these tests seem wrong just looking at them. Could you make them more realistic by using a more correct locale? Perhaps even some that we typically see/use within NI products (e.g. `de-DE`, `fr-FR`, etc).